third_party/gpus/cuda/BUILD.tpl

```diff
@@ -58,10 +58,8 @@ cc_library(
     linkopts = [
         "-ldl",
         "-lpthread",
-    ] + select({
-        "@//tensorflow:darwin": [],
```

Comment:
Ah, sorry, you're right. I was building from within the workspace of TF serving. I'll investigate this further.